x/ibc/04-channel/types/msgs.go

```diff
@@ -386,7 +386,7 @@ func (msg MsgPacket) GetSigners() []sdk.AccAddress {
 
 // Type implements sdk.Msg
 func (msg MsgPacket) Type() string {
-	return "ics04/opaque"
+	return "recv_packet"
```

Comment:
shouldn't this just be `packet`?

Reply:
I think usage of `recv` is a bit more clear. I wouldn't mind renaming `MsgPacket` to `MsgRecvPacket`

Reply:
ok from my end. @AdityaSripal  @cwgoes ?

Reply:
Yes I think `MsgPacket` should become `MsgRecvPacket` as well

Reply:
I can update that in a followup pr to keep things isolated

Reply:
that's fine